build-test.xml

```diff
@@ -9938,6 +9938,24 @@ org.apache.catalina.tribes.level = FINE]]>
 		</if>
 	</target>
 
+	<target name="deploy-maven-module">
+		<execute dir="${module.base.dir}">
+			mvn clean package
+		</execute>
+
+		<copy
+			flatten="true"
+			todir="${liferay.home}/deploy/"
+		>
+			<fileset
+				dir="${module.base.dir}"
+				includes="**/target/*.war"
+			/>
+		</copy>
+
+		<delete file="${module.base.dir}/target" />
+	</target>
```

Comment:
@Kvale1733, for the code `<delete file="${module.base.dir}/target" />`. Refer to https://ant.apache.org/manual/Tasks/delete.html, we should use **dir** property, instead of using **file** property. Please check if the folder can be deleted successful after running the test.